### PR TITLE
Added fix for handling one edge case for appinfo and removed the Hosted On Option.

### PR DESCRIPTION
**Pull request by Jonsy13**
Signed-off-by: Jonsy13 <vedant.shrotria@chaosnative.com>

## Proposed changes

This PR will add a fix for handling one edge case for app info ( when chaosengine doesn't have appinfo ) and removed the Hosted On Option.

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [X] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [X] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/views/WorkflowDetails/WorkflowTable/TableData.tsx [hunk 1]
```diff
@@ -39,47 +39,36 @@ const TableData: React.FC<TableDataProps> = ({
...
+          {data.finishedAt !== ''
+            ? (
+                (parseInt(data.finishedAt, 10) - parseInt(data.startedAt, 10)) /
+                60
+              ).toFixed(1)
+            : (
+                (new Date().getTime() / 1000 - parseInt(data.startedAt, 10)) /
+                60
+              ).toFixed(1)}{' '}
+          minutes
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/WorkflowDetails/WorkflowTable/TableData.tsx:58]: translation needed

**Review by amityt — APPROVED** [review]

File: litmus-portal/frontend/src/views/WorkflowDetails/WorkflowNodeInfo/index.tsx [hunk 2]
```diff
@@ -132,18 +133,17 @@ const WorkflowNodeInfo: React.FC<WorkflowNodeInfoProps> = ({
...
               {isAppInfoVisible && (
                 <Typography className={classes.textMargin}>
-                  {embeddedYAMLString &&
-                    Object.keys(
-                      YAML.parse(embeddedYAMLString).spec.appinfo
-                    ).map((key, index) => (
+                  {Object.keys(YAML.parse(embeddedYAMLString).spec.appinfo).map(
+                    (key, index) => (
                       <div key={index.toString()}>
                         <strong>{key} :</strong>
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/WorkflowDetails/WorkflowNodeInfo/index.tsx:139]: use a style={{fontWeight: 'bold'}} in the above div @Jonsy13

**Review by S-ayanide — APPROVED** [review]

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/views/WorkflowDetails/WorkflowNodeInfo/index.tsx (hunk 2 above)
Comment (Jonsy13) [reply litmus-portal/frontend/src/views/WorkflowDetails/WorkflowNodeInfo/index.tsx:139]: I don't think there is a problem with this, It helps to develop the code continuously without going to CSS file just for one CSS property. If it were 2 properties or more I would have made 1 CSS class, but here I don't it is necessary.

File: litmus-portal/frontend/src/views/WorkflowDetails/WorkflowTable/TableData.tsx (hunk 1 above)
Comment (Jonsy13) [reply litmus-portal/frontend/src/views/WorkflowDetails/WorkflowTable/TableData.tsx]: done

File: litmus-portal/frontend/src/views/WorkflowDetails/WorkflowNodeInfo/index.tsx (hunk 2 above)
Comment (ishangupta-ds) [reply litmus-portal/frontend/src/views/WorkflowDetails/WorkflowNodeInfo/index.tsx:139]: please remove the jsx @Jonsy13 no need of any class provide an inline style above to this `<div key={index.toString()}>` please do not add extra class.

Comment (Jonsy13) [reply litmus-portal/frontend/src/views/WorkflowDetails/WorkflowNodeInfo/index.tsx:139]: If I add an inline style to `<div key={index.toString()}>`, then the key-value pair both will be bold, which is not the design requirement.

Comment (ishangupta-ds) [reply litmus-portal/frontend/src/views/WorkflowDetails/WorkflowNodeInfo/index.tsx:139]: sure then this is good.

**Review by ishangupta-ds — APPROVED** [review]
LGTM

**Pull request merged**